cdap-api/src/main/java/co/cask/cdap/api/schedule/ScheduleBuilder.java

```diff
@@ -140,8 +145,39 @@
    * @param datasetName the name of the dataset in the specified namespace of the app
    * @param numPartitions the minimum number of new partitions added to the dataset to trigger the schedule
    * @return this {@link ScheduleBuilder}
+   * @deprecated since 4.3.0. Please use {@link ScheduleBuilder#setTriggerOnPartitions(String, String, int)} instead.
```

Comment:
This method was not present in 4.2, no need to deprecate or return `ScheduleCreationSpec`